dask_bigquery/core.py

```diff
@@ -66,6 +67,8 @@ def bigquery_read(
       Name of the BigQuery project.
     read_kwargs: dict
       kwargs to pass to read_rows()
+    arrow_options: dict
+      kwargs to pass to record_batch.to_pandas() when converting from pyarrow to pandas
```

Comment:
Maybe add a link to https://arrow.apache.org/docs/python/generated/pyarrow.RecordBatch.html#pyarrow.RecordBatch.to_pandas, so people can easily look up those kwargs.